backend/src/store/mysql.js

```diff
@@ -1,50 +1,87 @@
 const mysql = require('mysql');
+const config = require('../config/index');
 
-const { config } = require('../config/index');
-
-const dbconf = {
+const dbconf_pro = {
     host: config.mysql.host,
     user: config.mysql.user,
     password: config.mysql.password,
-    database: config.mysql.database,
+    database: config.mysql.name,
+};
+
+const dbconf_dev = {
```

Comment:
i think we don't need a second declaration of the environment settings because we have in the environment variables the correct connection to use